packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
```

Comment:
+1 I think it is likely the background color will need to be theme-able, which would probably require the box shadow to theme